lib/matplotlib/tests/test_axes.py

```diff
@@ -2902,6 +2902,24 @@ def test_errorbar_with_prop_cycle():
     ax.errorbar(x=[2, 4, 10], y=[6, 4, 2], yerr=0.5)
 
 
+@image_comparison(baseline_images=['errorbar_offsets'],
```

Comment:
Small additional request: We are trying to avoid adding more test images to the repository (because they cumulatively contribute a lot of disk space...), and this specific test looks like it could be written using the `check_figures_equal` decorator instead.  You can find some examples already in the repository, but basically the idea would be to compare code that generates the figure using errorevery and code that generates the figure using manual downsampling of the errorbars.
If you get this to work, please also squash the baseline images currently present out of the history.

Reply:
Funny, I was thinking about that very problem earlier today.  I'll give it a shot.

Reply:
See also https://github.com/matplotlib/matplotlib/pull/12552 for some writeup about that point.